Source/Core/DiscIO/Volume.h

```diff
@@ -66,7 +66,9 @@ class Volume
   virtual Partition GetGamePartition() const { return PARTITION_NONE; }
   virtual std::optional<u32> GetPartitionType(const Partition& partition) const { return {}; }
   std::optional<u64> GetTitleID() const { return GetTitleID(GetGamePartition()); }
-  virtual std::optional<u64> GetTitleID(const Partition& partition) const { return {}; }
+  // TODO: change this back to "return {}" once this bug is fixed:
```

Comment:
These comments can probably be removed, given std::nullopt is more than acceptable here.

`std::nullopt` should be preferred anyways, since it allows implementations to avoid unnecessarily zeroing out the internal buffer of the optional ([example](https://godbolt.org/z/esrrbq))

`ReadSwappedAndShifted, and GetPartitionType() should probably be converted over to using std::nullopt as well.